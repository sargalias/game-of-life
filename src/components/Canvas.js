import React from 'react';

const Canvas = () => (
  <div id="canvas-container">
    <canvas id="canvas" />
  </div>
);

export default Canvas;
