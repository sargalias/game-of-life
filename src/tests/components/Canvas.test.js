import React from 'react';
import { shallow } from 'enzyme';
import Canvas from '../../components/Canvas';

test('should render Canvas correctly', () => {
  const wrapper = shallow(<Canvas />);
  expect(wrapper).toMatchSnapshot();
});
