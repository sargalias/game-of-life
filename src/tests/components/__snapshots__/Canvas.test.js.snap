// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Canvas correctly 1`] = `
<div
  id="canvas-container"
>
  <canvas
    id="canvas"
  />
</div>
`;
